### ISSUE TITLE
Parameter to add seek in increments

### ISSUE THREAD

**Issue by 2V3EvG4LMJFdRe** [po5/evafast#11]
First of all thanks for your work on this script. 

I'm coming from https://github.com/mpv-player/mpv/issues/11589 and it seems like a change to this script's core could implement the type of seek that streaming services use (by an exponential rate). The problem for my use case is there's no `speed_cap` big enough that it will allow to seek to the end of a 01:30:00 video in a reasonable amount of time - it's actually faster to just use `seek 3`.

I was wondering if you could add a parameter and function like `seek_increase` which would add a value to `seek` at a fixed rate through `speed_interval` (or a separate `seek_interval` parameter). This in conjunction with the forward feature of this script, which is visually a very nice touch, could create a really robust seeking control.

**Comment by po5** [comment]
Hey,
This script is meant for fast-forwarding, seeking behavior is basic.
The idea being that evafast doesn't require extra keys, I don't see how this fits without shadowing the fastforward feature.
Should be its own script, wouldn't be incompatible with evafast so long as you keep it on its own key, and would work just as well.
Which services implement this seeking behavior? I've never seen it.

**Comment by 2V3EvG4LMJFdRe** [comment]
The idea is that it would be the same `evafast` key with an added function, no extra keys required nor need to disable the rewind. Presuming the function to fast forward in increments until the key is released is there, it would take some edits to add in a `seek` feature, but I understand that it might be out of scope of your design and could become hard to maintain.

I just tried Netflix and you're right that it's not there. That's so strange, I swear at least one streaming service had this behaviour. It makes a lot of sense to me UX-wise, I felt pretty awkward testing Netflix just now taking forever to go back to the beginning of the video. The more you're holding the key, the more likely it is you want to advance. Someone in the issue I mentioned has now created a lua script that does it if you wanna try it out - sadly for my use case it happens not to work on IINA and I'm waiting on their end to see what's going on. I'll keep the issue open since I think it could scale up this script in a positive way, but of course feel free to close it if it's not of interest.
